src/helpers/createHistoryRecord.js

```diff
@@ -1,13 +1,12 @@
-const createHistoryRecord = (url) => {
-	window.history.pushState(
-		{
-			url: url || window.location.href.split(window.location.hostname)[1],
-			random: Math.random(),
-			source: 'swup'
-		},
-		document.title,
-		url || window.location.href.split(window.location.hostname)[1]
-	);
+const createHistoryRecord = (url, customData = {}) => {
+	url = url || window.location.href.split(window.location.hostname)[1];
```

Comment:
While we are at it, couldn't we use swups `getCurrentUrl()` here?
```js
import getCurrentUrl from "./getCurrentUrl";
const createHistoryRecord = (url, customData = {}) => {
	url = url || getCurrentUrl();
    // ...
}
```

Reply:
Also, I always found `getCurrentUrl` always a bit confusing, naming-wise. How about we change it to `getCurrentPath` and add an alias for backwards-compatibility, what do you think?

Reply:
Done :) I'd prefer not to touch the naming, just maintenance-wise.

Reply:
Hmm. I really find it confusing that we are using `getCurrentUrl()`, but then we also use `resolvePath` on it. We should probably either 

1. rename `resolvePath()` to `resolveUrl()`
2. or rename `getCurrentUrl()` to  `getCurrentPath()`

Since the thing actually _is_ a relative URL composed by `pathname` + `searchParams`, and also for backwards-compatibility, I would tend to rename the new option #1 then. What do you think?

Reply:
In that case, I also prefer option 1, renaming the new option `resolveUrl`. Could you do that for the main repo and the docs? And the scroll plugin, I guess, since that's already using it.